pkg/api/api.go

```diff
@@ -65,7 +65,7 @@ func (s *APIServer) Init() error {
 	router.HandleFunc("POST /api/forgot-password", makeHTTPHandlerFunc(s.authHandlers.ForgotPassword))
 	router.HandleFunc("POST /api/change-password", makeHTTPHandlerFunc(s.authHandlers.ChangePassword))
 	router.HandleFunc("POST /api/users", makeHTTPHandlerFunc(s.authHandlers.RegisterUser))
-	router.HandleFunc("POST /api/users/{id}/verify-email", makeHTTPHandlerFunc(s.authHandlers.VerifyEmail))
+	router.HandleFunc("GET /api/users", makeHTTPHandlerFunc(s.authHandlers.VerifyEmailOnTemplate))
```

Comment:
I feel this new api route name is unclear. A better name to keep things RESTful could be something like:

```suggestion
router.HandleFunc("GET /api/users/verify", makeHTTPHandlerFunc(s.authHandlers.VerifyEmailOnTemplate))
```